instrumentation/kafka/kafka-clients/README.md

```diff
@@ -3,4 +3,231 @@
 | System property | Type | Default | Description |
 |---|---|---|---|
 | `otel.instrumentation.kafka.experimental-span-attributes` | Boolean | `false` | Enable the capture of experimental span attributes. |
-| `otel.instrumentation.kafka.client-propagation.enabled` | Boolean | `true` | Enables remote context propagation via Kafka message headers. |
\ No newline at end of file
+| `otel.instrumentation.kafka.client-propagation.enabled` | Boolean | `true` | Enables remote context propagation via Kafka message headers. |
+
+# Kafka Metrics
```

Comment:
This whole new section only applies to the library instrumentation, right? (the agent will not automatically inject the `OpenTelemetryKafkaMetrics` class)
How about we put it into the library instrumentation module README?